README.md

```diff
@@ -89,7 +91,6 @@ Attribute              | Type          | Default                | Description
 ---                    | ---           | ---                    | ---
 `remoteUrl`            | *String*      |  **Required**          | PATCH server URL
 `callback`             | *Function*    |                        | Called after initial state object is received from the server (NOT necessarily after WS connection was established)
-`obj [readonly]`      | *Object*      | `{}`                    | Your initial state object, **please read notes below**.
```

Comment:
Please add this description bellow, as in Properties section it says "see above" and there is nothing above ;)